src/app/settings/admin/interface/template.html

```diff
@@ -92,10 +92,12 @@
              class="entry-label">
           <span>Enable OIDC Kubeconfig</span>
           <div class="km-icon-info km-pointer"
-               matTooltip="Use OIDC provider as a proxy for kubeconfig download."></div>
+               matTooltip="Use OIDC provider as a proxy for kubeconfig download. Requires OIDCKubeCfgEndpoint feature gate to be enabled."></div>
         </div>
         <mat-checkbox [(ngModel)]="settings.enableOIDCKubeconfig"
+                      [disabled]="!isOIDCKubeCfgEndpointEnabled"
                       (change)="onSettingsChange()"
+                      [matTooltip]="!isOIDCKubeCfgEndpointEnabled ? 'OIDCKubeCfgEndpoint feature gates needs to enabled.' : null"
```

Comment:
It may be not clear where the flag can be found.

Reply:
Still I am not sure about using internal names like `OIDCKubeCfgEndpoint` or `KubermaticConfiguration` in the UI. Not all end users will know them, it's clear for devs but it might be not for all.

Reply:
@cschieder what do you think, how it should look like in your opinion?

Reply:
What'd you suggest then? Just keep it disabled without any additional info?

Reply:
Could we call it by some name that describes it's functionality, in general maybe? (both here and in the documentation) And then say "Feature is disabled. Check out our documentation about xxx to learn more." Or the like?

Reply:
Hmm, will [this](https://docs.kubermatic.com/kubermatic/master/tutorials_howtos/oidc_provider_configuration/share-_clusters_via_delegated_oidc_authentication/) be adequate? Thought we had support for anchor tags in our docs but I think we don't 🤷‍♂️

Reply:
I am not sure if the link inside the tooltip will actually work.